src/Illuminate/Notifications/RoutesNotifications.php

```diff
@@ -20,6 +20,8 @@ public function notify($instance)
 
     /**
      * Get the notification routing information for the given driver.
+     * In case of an empty return value, the notification will
+     * not be sent.
```

Comment:
hmmm, just try to send an email to someone whom has no email on record :) .. it will fire off just like it would normally (which it should because anything else would be annoying) but since there is no email there is no-one to send to, so no email will be sent. This is all working as one would expect for a production system. 

But when you play with it and you're trying to debug it, the brain somehow expects an error message if the email is not delivered, at least mine did in this case. Those extra two lines should hopefully save someone a headache trying to figure out why emails aren't showing up :)